scripts/hostcfgd

```diff
@@ -1440,6 +1441,75 @@ class MgmtIfaceCfg(object):
         self.mgmt_vrf_enabled = enabled
 
 
+class SyslogCfg:
+    SYSLOG_RATE_LIMIT_INTERVAL = 'rate_limit_interval'
+    SYSLOG_RATE_LIMIT_BURST = 'rate_limit_burst'
+    HOST_KEY = 'GLOBAL'
+
+    # syslog conf file path in docker
+    SYSLOG_CONF_PATH = '/etc/rsyslog.conf'
+
+    # Regular expressions to extract value from rsyslog.conf
+    INTERVAL_PATTERN = '.*SystemLogRateLimitInterval\s+(\d+).*'
+    BURST_PATTERN = '.*SystemLogRateLimitBurst\s+(\d+).*'
+
+    def __init__(self):
+        self.current_interval, self.current_burst = self.parse_syslog_conf()
+
+    def syslog_update(self, data):
+        """Update syslog related configuration
+
+        Args:
+            data (dict): CONFIG DB data: {<field_name>: <field_value>}
+        """
+        new_interval = '0'
+        new_burst = '0'
+        if data:
+            new_interval = data.get(self.SYSLOG_RATE_LIMIT_INTERVAL, '0')
+            new_burst = data.get(self.SYSLOG_RATE_LIMIT_BURST, '0')
+
+        if new_interval == self.current_interval and new_burst == self.current_burst:
+            syslog.syslog(syslog.LOG_INFO, 'Syslog rate limit configuration does not change, ignore it')
```

Comment:
I'd recommend to drop this info log.  In our environment, we already have a lot of logs.  :)